packages/patrol_cli/lib/src/commands/build_android.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:path/path.dart' show basename;
+import 'package:patrol_cli/src/android/android_test_backend.dart';
+import 'package:patrol_cli/src/base/exceptions.dart';
+import 'package:patrol_cli/src/base/extensions/core.dart';
+import 'package:patrol_cli/src/base/logger.dart';
+import 'package:patrol_cli/src/crossplatform/app_options.dart';
+import 'package:patrol_cli/src/dart_defines_reader.dart';
+import 'package:patrol_cli/src/pubspec_reader.dart';
+import 'package:patrol_cli/src/runner/patrol_command.dart';
+import 'package:patrol_cli/src/test_finder.dart';
+
+class BuildAndroidCommand extends PatrolCommand {
+  BuildAndroidCommand({
+    required TestFinder testFinder,
+    required DartDefinesReader dartDefinesReader,
+    required PubspecReader pubspecReader,
+    required AndroidTestBackend androidTestBackend,
+    required Logger logger,
+  })  : _testFinder = testFinder,
+        _dartDefinesReader = dartDefinesReader,
+        _pubspecReader = pubspecReader,
+        _androidTestBackend = androidTestBackend,
+        _logger = logger {
+    usesTargetOption();
+    usesBuildModeOption();
+    usesFlavorOption();
+    usesDartDefineOption();
+    usesLabelOption();
+    usesWaitOption();
+
+    usesAndroidOptions();
+  }
+
+  final TestFinder _testFinder;
+  final DartDefinesReader _dartDefinesReader;
+  final PubspecReader _pubspecReader;
+  final AndroidTestBackend _androidTestBackend;
+
+  final Logger _logger;
+
+  @override
+  String get name => 'android';
+
+  @override
+  String get description => 'Build app for integration testing on Android.';
+
+  @override
+  Future<int> run() async {
+    final targetArg = stringsArg('target');
+    if (targetArg.isEmpty) {
+      throwToolExit('No test target specified');
+    } else if (targetArg.length > 1) {
+      throwToolExit('Only one test target can be specified');
+    }
+    final target = _testFinder.findTest(targetArg.single);
+    _logger.detail('Received test target: $target');
+
+    final config = _pubspecReader.read();
+    final flavor = stringArg('flavor') ?? config.android.flavor;
+    if (flavor != null) {
+      _logger.detail('Received Android flavor: $flavor');
+    }
+
+    final packageName = stringArg('package-name') ?? config.android.packageName;
+
+    final displayLabel = boolArg('label');
+
+    final customDartDefines = {
+      ..._dartDefinesReader.fromFile(),
+      ..._dartDefinesReader.fromCli(args: stringsArg('dart-define')),
+    };
+    final internalDartDefines = {
+      'PATROL_WAIT': defaultWait.toString(),
```

Comment:
~Where does the `defaultWait` come from?~

I see it's from `PatrolCommand`. But what's the wait? Wait for/before what? And in what unit is it specified? Please add dartdoc for that field